docs/detections/alerts-add-to-cases.asciidoc

```diff
@@ -16,10 +16,13 @@ To add alerts to a new case:
 . Do one of the following:
 ** To add a single alert to a case, select the *More actions* menu (*...*) in the Alerts table or **Take action** in the alert details flyout, then select *Add to a new case*.
 ** To add multiple alerts, select the alerts, then select *Add to a new case* from the *Bulk actions* menu.
-. In the **Create new case** flyout, give your case a name, add relevant tags (optional), assign a severity level, and include a case description.
+. Give the case a name, assign a severity level, and provide a description. You can use
+https://www.markdownguide.org/cheat-sheet[Markdown] syntax in the case description.
```

Comment:
We might want to link to [this page](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) instead; that's what the UI links to, when you click the Markdown icon to get "Syntax help":
<img width="600" alt="image" src="https://user-images.githubusercontent.com/87339667/200603647-f7071e04-b3d8-44dc-8e17-014a90f5abd8.png">

It'd probably be OK to omit the link, too — Markdown seems pretty standard by now, so we prob don't need to tell users what it is. For comparison, Kibana docs have lots of references to Markdown without links.
```suggestion
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax[Markdown] syntax in the case description.
```

Reply:
Good eye, thanks @joepeeples ! I'm leaning towards keeping it for now but would not be opposed to removing it down the line since the UI provides a link too.